keyboards/ut472/rules.mk

```diff
@@ -67,4 +30,3 @@ BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 RGBLIGHT_ENABLE = yes
-TAP_DANCE_ENABLE = no      # Enable tap dance feature
```

Comment:
Why the deletion for `TAP_DANCE_ENABLE`?

Reply:
Not sure, but it defaults to `no` so there should be no behaviour change. I'll put it back anyway.